src/EditorFeatures/TestUtilities/Diagnostics/DiagnosticTaggerWrapper.cs

```diff
@@ -79,10 +79,7 @@ public ITaggerProvider TaggerProvider
                 {
                     WpfTestRunner.RequireWpfFact($"{nameof(DiagnosticTaggerWrapper<TProvider, TTag>)}.{nameof(TaggerProvider)} creates asynchronous taggers");
 
-                    if (typeof(TProvider) == typeof(DiagnosticsSquiggleTaggerProvider)
-                        || typeof(TProvider) == typeof(DiagnosticsSuggestionTaggerProvider)
-                        || typeof(TProvider) == typeof(DiagnosticsClassificationTaggerProvider)
-                        || typeof(TProvider) == typeof(InlineDiagnosticsTaggerProvider))
+                    if (typeof(TProvider) == typeof(InlineDiagnosticsTaggerProvider))
```

Comment:
Will the inline diagnostics tagger also eventually move down to the platform layer?

Reply:
No talks currently.  But it would make sense to be automatic with LSP pull diags.  We can bring up with them.  For now though, it's fine for us to continue providing this on our end :)